### PR TITLE
Bookies interface with vuetify

### PR DESCRIPTION
**Pull request by dcdiennea**

### CODE REVIEW

File: web/src/main/bvmui/src/components/Card.vue [hunk 1]
```diff
@@ -1,28 +1,44 @@
...
-    </div>
+    <v-card class="mx-auto" :color="statuscolor" raised dark @click="$emit('click', $event)">
+        <v-list-item two-line>
+        <v-list-item-content>
+            <v-list-item-title class="headline">{{data.description}}</v-list-item-title>
+            <v-list-item-subtitle>{{$library.formatDate(data.lastScan)}}</v-list-item-subtitle>
+        </v-list-item-content>
+        </v-list-item>
+        <v-card-text>
+            <p class="text-center display-3">{{$library.formatPercent(data.totalDiskSpace - data.freeDiskSpace, data.totalDiskSpace)}}&#37;</p>
```
Comment (eolivelli) [inline web/src/main/bvmui/src/components/Card.vue]: unfortunately totalDiskSpace and totalDiskSpace are not always available.
I won't put this information in the foreground.
The most useful info are:
- the ID of the bookie (hostname:port)
- The status: available, readonly, down

File: web/src/main/bvmui/src/components/Card.vue [hunk 2]
```diff
@@ -1,28 +1,44 @@
 <template>
-    <div class="bvm-card" @click="$emit('click', $event)">
-        <div class="bvm-card-description">{{data.description}}</div>
-        <div class="bvm-card-status">
-            <h1 >{{data.state}}</h1>
```
Comment (eolivelli) [inline web/src/main/bvmui/src/components/Card.vue]: this should be explicitly stated, using colours is not enough expecially for an operator that is the first time that uses the tool

File: web/src/main/bvmui/src/lib/library.js [hunk 3]
```diff
@@ -24,5 +24,18 @@ export default {
...
+    formatDate(ts){
+        if (ts){
+            const months = ["JAN", "FEB", "MAR","APR", "MAY", "JUN", "JUL", "AUG", "SEP", "OCT", "NOV", "DEC"];            
+            let current_datetime = new Date()
+            let formatted_date = current_datetime.getDate()  + " "
+            + (months[current_datetime.getMonth()]) + " " 
+            + current_datetime.getFullYear() + " " 
+            + current_datetime.getHours() + ":" 
+            + (current_datetime.getMinutes() < 10 ? "0" + current_datetime.getMinutes() : current_datetime.getMinutes()) 
+            console.log("formatted_date",formatted_date);
```
Comment (eolivelli) [inline web/src/main/bvmui/src/lib/library.js]: remove debug please

**Comment by mino181295** [comment]
Nice done! 
After you resolved the comments for me it's +1

File: web/src/main/bvmui/src/lib/library.js (hunk 3 above)
Comment (dcdiennea) [reply web/src/main/bvmui/src/lib/library.js]: removed

File: web/src/main/bvmui/src/components/Card.vue (hunk 2 above)
Comment (dcdiennea) [reply web/src/main/bvmui/src/components/Card.vue]: ok

File: web/src/main/bvmui/src/components/Card.vue (hunk 1 above)
Comment (dcdiennea) [reply web/src/main/bvmui/src/components/Card.vue]: ok

**Pull request merged**